apps/web/public/static/locales/en/common.json

```diff
@@ -2042,6 +2042,7 @@
   "organization_banner_title": "Manage organizations with multiple teams",
   "set_up_your_organization": "Set up your organization",
   "set_up_your_platform_organization": "Set up your platform",
+  "platform_account_warning": "Do not use your personal Cal.com account or account used to manage teams and organizations. Create a new Cal.com account specifically for setting up the Platform and only then continue on this page.",
```

Comment:
@supalarry I would try to use a message that is more assertive, I know this is not what we intend but imho I think this message comes off as a bit aggressive or maybe the tone that I'm reading it in is that way. But what I want to say is maybe we start off by giving the user some context by saying something like `since platform and all the other plans (personal, teams and orgs) use two separate infrastructures we advice you to create a new account`